build.gradle.kts

```diff
@@ -70,7 +70,7 @@ dependencies {
   testImplementation("org.mockito:mockito-inline:5.2.0")
   testImplementation("io.swagger.parser.v3:swagger-parser:2.1.18")
   testImplementation("org.springframework.security:spring-security-test")
-  testImplementation("com.github.tomakehurst:wiremock-jre8-standalone:3.0.1")
+  testImplementation("org.wiremock:wiremock:3.3.1")
```

Comment:
other teams have made this switch [e.g.](https://github.com/ministryofjustice/hmpps-prisoner-search/commit/c002b7b3bfd28e8069b96563b75ab3fc0c535e18#diff-2bdbd18573bf706cac582086e5b2a977ef2d83dbfad6957bb8e7cb3952a5cc14R42)